jenkins/test-history/gen_history

```diff
@@ -22,6 +22,10 @@ set -o nounset
 readonly bucket="kubernetes-test-history"
 readonly jsonpath="gs://${bucket}/logs/$(date +%F).json"
 
+# Copy buckets.yaml so the Docker container can access it
+cp ../../buckets.yaml .
```

Comment:
I guess this will be current each time it runs we download the latest version of the repo before running? I'd prefer we not depend on this.